searxng/searxng
# Convert the infinite_scroll plugin as a preference

## What does this PR do?

* oscar theme: done
* simple theme: done except keyboard navigation

TODO:
* [x] in the category images, the scrollbar stay at the bottom of the page
* [x] check `searxng.http` still works in the generic cases
* [x] vim keyboard navigation doesn't work  --> https://github.com/searxng/searxng/issues/836
* [x] ~(optional / debatable) convert on the fly the preferences from the old format = enable infinite scroll when the cookie `enabled_plugins` contains `searx.plugins.infinite_scroll`.~

## Why is this change important?

* Implement the infinite scroll for the simple theme

## How to test this PR locally?

* `make themes`
* with different browser, at least Firefox and Chrom* : 
   * in the simple theme, and in the oscar theme:
     * check a request in the general category
     * check a request in the images category
     * check a request `!general !images time`

## Author's checklist



## Related issues

Part of #769
